cep-9999.md

```diff
@@ -0,0 +1,165 @@
+
+## Precedence and role in the pinning resolution process
+
+This file is not meant to replace the `run_exports` metadata already present in the package archives. It merely presents that information in a more convenient way. 
+`conda-build`-like clients are free to use either the `run_exports` metadata in the archives or the one in `run_exports.json`, since they should be equivalent.
+Special keys like `pin_run_as_build` will keep their behavior, since `run_exports.json` does not add a new level of precedence in the pinning resolution process. Again, it's an equivalent source for information already present in the archives.
+
+This means that `run_exports.json` should not be patched (like it is done with `repodata.json`);
+it should always reflect the metadata present in the archives. 
+If at some point this decision needs to be reverted, it will involve a change in the schema version to ensure backwards compatibility.
+
+## Specification
+
+The schema of `run_exports.json` would mimic the `repodata.json` structure whenever possible:
+
+* `info`: metadata about the platform, architecture, and version of the `run_exports.json` schema.
+* `packages`: map of `.tar.bz2` filenames to `run_exports` metadata `dict`.
+* `packages.conda`: map of `.conda` filenames to `run_exports` metadata `dict`.
+
+Each `run_exports` metadata `dict` can contain the following fields; each field accepts a list of strings (conda-build specs).
+
+- `weak`
+- `strong`
+- `weak_constrains`
+- `strong_constrains`
+- `noarch`
+
+```json
+{   
+    "info": {
+        "platform": "string",
+        "arch": "string",
+        "subdir": "string",
+        "version": 0
+    },
+    "packages": {
+        "package-version-build.tar.bz2": {
+            "run_exports": {
+                "noarch": [
+                    "string",
+                ],
+                "strong": [
+                    "string",
+                ],
+                "strong_constrains": [
+                    "string",
+                ],
+                "weak": [
+                    "string",
+                ],
+                "weak_constrains": [
+                    "string",
+                ],
+            }
+        }
+    },
+    "packages.conda": {
+        "package-version-build.conda": {
+            "run_exports": {
+                "noarch": [
+                    "string",
+                ],
+                "strong": [
+                    "string",
+                ],
+                "strong_constrains": [
+                    "string",
+                ],
+                "weak": [
+                    "string",
+                ],
+                "weak_constrains": [
+                    "string",
+                ],
+            }
+        }
+    }
+}
+```
+
+If a package does not define `run_exports`, the corresponding entry in `packages` or `packages.conda` will be an empty `run_exports` item:
```

Comment:
If we decide to adopt BCP 14 conventions, I suggest replacing "...will be an empty..." with "...MUST be an empty...".